src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -527,7 +527,8 @@ private JabRefPreferences() {
             if (new File("jabref.xml").exists()) {
                 importPreferences(Path.of("jabref.xml"));
             }
-        } catch (JabRefException e) {
+        } catch (
+                JabRefException e) {
```

Comment:
OMG, our formatter...

Would it be possible to unstage these lines from the commit?